chapter3/chapter3.tex

```diff
@@ -106,240 +111,261 @@ \section{What is a random variable?}
+about the underlying sample space but only about certain quantities
+captured by the RVs. Therefore, we define a probability distribution over the
+random variables of interest. Since by definition the distribution has to fulfil conditions that would allow us to interpret it
+as a probability measure on a probability space, we can use it without worrying
+about the underlying sample space too much. \chris{hmmm, not sure about this statement}\philip{I tried to formulate this a bit
+more carefully. What do you think?}
 
 Joint distributions are of particular interest because they make it possible to recover the probability distribution of each individual
-RV as well as of smaller joint distributions. The process by which this can be accomplished is known as \textbf{marginalization}. Say
-we are given a joint distribution $ P_{XY} $. Now how can we determine the probability that $ X=x $? Our sample space is carved
+RV as well as of smaller joint distributions. The process by which this recovery can be accomplished is known as \textbf{marginalization}. Say
+we are given a joint distribution $ P_{XY} $. How can we determine the probability that $ X=x $? Our sample space is carved
 up by the values that $ X $ and $ Y $ can take on. We are interested in the probability of the subset $ E = \{\omega|X(\omega)=x\} $.
-Assuming that $ Y $ can take on $ n $ values, let us define $ E_{i} = \{\omega| Y(\omega) = y_{i}\} $ for $ 1 \leq i \leq n $.
-Some (possibly all) of the $ E_{i} $ will overlap with $ E $. Thus we can partition $ E $ into $ E\cap E_{1}, \ldots, E \cap E_{n} $.
-Now by countable additivity we simply need to sum up the probabilities of these intersections. Observe that those probabilities
-correspond exactly to $ P_{XY}(X=x,Y=y_{1}), \ldots, P_{XY}(X=x,Y=y_{1}) $. Hence, we get to the following equality:
+Assuming that $ Y $ can take on $ n $ values $y_1,y_2,\ldots,y_n$, let us define $ F_{i} = \{\omega| Y(\omega) = y_{i}\} $ for $ 1 \leq i \leq n $.
+Some (possibly all) of the $ F_{i} $ will overlap with $ E $. Thus we can partition $ E $ into $ E\cap F_{1}, \ldots, E \cap F_{n} $.
+By countable additivity, we simply need to sum up the probabilities of these intersections. Observe that those probabilities correspond exactly to $ P(X=x,Y=y_{1}), \ldots, P(X=x,Y=y_{1}) $. Hence, we get to the following equality:
 
 \begin{equation} \label{simpleMarginal}
-P_{X}(X=x) = \overset{n}{\underset{i=1}{\sum}} P_{XY}(X=x,Y=y_{i}) 
+P(X=x) = \overset{n}{\underset{i=1}{\sum}} P(X=x,Y=y_{i}) 
 \end{equation}
 
-Now if we set $ X = (Z_{1}, \ldots, Z_{m}) $, we can recover $ P_{Z_{1}^{m}} $ from $ P_{YZ_{1}^{m}} $ in the same way. Likewise,
-if we set $ Y = (Z_{1}, \ldots, Z_{m}) $ we get the generalization of \ref{simpleMarginal}. We will assume that each $ Z_{j} $
+If we set $ X = (Z_{1}, \ldots, Z_{m}) $, we can recover $ P_{Z_{1}^{m}} $ from $ P_{Z_{1}^{m}Y} $ in the same way.\chris{so?}\philip{Sorry, I do not understand your comment here.} Likewise,
```

Comment:
Yes, it is obvious, but I am assuming here that people have never worked with random variables before and do not know that it is "allowed" to a assign a random vector to a new variable. That's why I would rather leave it in.